client/src/main.h

```diff
@@ -234,6 +234,12 @@ struct CursorState
 
   /* the guest's cursor position */
   struct CursorInfo guest;
+
+  /* the projected position after move, for app_handleMouseBasic only */
+  struct {
+    int x;
+    int y;
+  } projected;
```

Comment:
Please use `struct Point` as defined in `common/types.h`

Reply:
Fixed.